app/controllers/optOut/OptOutChooseTaxYearController.scala

```diff
@@ -48,41 +47,36 @@ class OptOutChooseTaxYearController @Inject()(val optOutChooseTaxYear: OptOutCho
   def show(isAgent: Boolean = false): Action[AnyContent] = auth.authenticatedAction(isAgent) {
     implicit user =>
       for {
-        availableOptOutTaxYear <- optOutService.getTaxYearsAvailableForOptOut()
-        submissionCountForTaxYear <- optOutService.getQuarterlyUpdatesCountForTaxYear(availableOptOutTaxYear)
+        optOutProposition <- optOutService.fetchOptOutProposition()
```

Comment:
With this more direct use of optOutProposition, optOutService.getTaxYearsAvailableForOptOut() is now only called from tests. Can this be removed?

Reply:
done, removed optOutService.getTaxYearsAvailableForOptOut()

Reply:
Cool.